### ISSUE TITLE
extras don't seem to work

### ISSUE THREAD

**Issue by reflog** [chriswalz/bit#4]
Hi!
First of all - bit seems really awesome! Love the new workflow.

Tried using the extras, having some issues:

```
➜ bit info
--- INFO ---
fork/exec /tmp/bit/git-extras/git-info.sh: text file busy
--- SUMMARY ---
fork/exec /tmp/bit/git-extras/git-summary.sh: text file busy
--- EFFORT ---
fork/exec /tmp/bit/git-extras/git-effort.sh: text file busy
```

```
➜ bit effort
git: 'effort' is not a git command. See 'git --help'.
Command may not exist exit status 1
```

**Comment by fsmiamoto** [comment]
I'm experiencing different but related issues:
```sh
$ bit info
...
--- SUMMARY ---
git: 'root' is not a git command. See 'git --help'.
...
--- EFFORT ---
/bin/sh: Line 2: git_extra_mktemp: Command not found
```
I supposed this is related to some of the provided scripts being dependent on other ones from `git-extras`.

**Comment by chriswalz** [comment]
This will get resolved in the week or so

**Comment by chriswalz** [comment]
mostly fixed by dbab04a39
